manifests/overlays/prod/configs/argo_cm/resource.inclusions

```diff
@@ -69,9 +69,6 @@
 - apiGroups:
   - argoproj.io
   kinds:
-  - CronWorkflow
-  - Workflow
-  - WorkflowTemplate
```

Comment:
Why remove these?

Reply:
Because they are added by the [line 88](https://github.com/AICoE/aicoe-cd/pull/109/files#diff-9f4c2716bbd7db75239fab641800396cR88)